lib/services/local/access_list_test.go

```diff
@@ -106,13 +106,113 @@ func TestAccessListCRUD(t *testing.T) {
 	require.NoError(t, err)
 	require.Empty(t, cmp.Diff(accessList1, accessList, cmpOpts...))
 
+	// There should be no access list members
+	members, err := service.GetAccessListMembers(ctx, accessList1.GetName())
```

Comment:
I think this test is definitely doing too much now.

Reply:
Working on trying to split this out.

Reply:
Thanks, Mike.  👍

Reply:
I split this into its own AccessListMembersCRUD function, let me know if that's looking alright.